client/src/components/LayoutComponents/Dashboard/NewMember.jsx

```diff
@@ -110,56 +123,147 @@ export default function AddNewMember() {
 	};
 
 	return (
-		<form
-			className={classes.root}
-			noValidate
-			autoComplete="off"
-			onSubmit={handleSubmit}
-		>
-			<div>
-				<TextField
-					required
-					name="name"
-					error={!empData.name.isValid}
-					autoFocus={true}
-					id="standard-required"
-					label="Name"
-					onChange={handleChange}
-					autoFocus={true}
-				/>
-				<TextField
-					required={true}
-					name="email"
-					error={!empData.email.isValid}
-					autoFocus={true}
-					id="standard-required"
-					label="Email"
-					onChange={handleChange}
-				/>
-				<TextField
-					required={true}
-					name="date"
-					id="date"
-					label="Hire date"
-					type="date"
-					defaultValue="2020-05-24"
-					className={classes.textField}
-					InputLabelProps={{
-						shrink: true,
-					}}
-					onChange={handleChange}
-				/>
-				<TextField
-					required={true}
-					name="position"
-					id="standard-required"
-					label="position"
-					onChange={handleChange}
-				/>
-				<Button size="medium" color="primary" variant="contained" type="submit">
-					ADD
-				</Button>
-			</div>
-		</form>
+		<Box component="div" bgcolor="info.main" mx={23} mt={6}>
+			<Grid container className={classes.root}>
+				<Grid item xs={6} alignContent="center">
+					<Box>
+						<Card className={classes.card}>
+							<CardMedia
+								className={classes.media}
+								image={require('../../../img/addEmp.png')}
+								title="img"
+							/>
+							<CardContent>
+								<Typography variant="h5" color="textSecondary" component="p">
+									نظام إدارة الموارد البشرية
+								</Typography>
+							</CardContent>
+						</Card>
+					</Box>
+				</Grid>
+				<Grid
+					item
+					xs={6}
+					spacing={1}
+					alignItems="flex-end"
+					justify="flex-start"
+					className={classes.gutterBottom}
+					justify="center"
+				>
+					<form autoComplete="off" onSubmit={handleSubmit}>
+						<div dir="rtl">
+							<Box
+								display="flex"
+								flexDirection="column"
+								alignContent="center"
+								m={6}
+							>
+								<Typography color="primary">املا بيانات الموظف</Typography>
+								<Box m={2}>
+									<TextField
+										className={classes.fullWidth}
+										required={true}
+										error={!empData.name.isValid}
+										autoFocus={true}
+										name="name"
+										id="standard-required"
+										placeholder="الاسم"
+										onChange={handleChange}
+										variant="filled"
+										InputProps={{
+											startAdornment: (
+												<InputAdornment position="start">
+													<Person />
+												</InputAdornment>
+											),
+										}}
+									/>
+								</Box>
+								<Box m={2}>
+									<TextField
+										className={classes.fullWidth}
+										required={true}
+										error={!empData.email.isValid}
+										autoFocus={true}
+										name="email"
+										id="standard-required"
+										placeholder="الايميل"
+										onChange={handleChange}

```

Comment:
what if the user enters a name in email input, I mean not email formatting, should that be okay?